app/security.py

```diff
@@ -47,7 +47,7 @@
     'X-Content-Security-Policy': CSP,
     'X-Frame-Options': 'DENY',
     'X-Content-Type-Options': 'nosniff',
-    'Referrer-Policy': 'no-referrer',
+    'Referrer-Policy': 'strict-origin-when-cross-origin',
```

Comment:
Is this the most secure it can be set to?

Reply:
there are more "secure" options, but this is a sensible option and also allows the sharing of the full referrer with our domain which some of the design system needs, this is a good guide on how to choose the right option: https://web.dev/referrer-best-practices/#:~:text=Consider%20setting%20a%20referrer%20policy,an%20extra%20layer%20of%20security.